coops-ui/src/components/dialogs/CancelPaymentErrorDialog.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <v-dialog v-model="dialog" width="45rem" persistent :attach="attach">
```

Comment:
Please add a `content-class` attribute to select this dialog.

Also please remove all `data-test-id` below where we already have an element id.

Reply:
data-test-ids need another discussion with QA

Reply:
> I have made comments regarding "data-test-id" attributes but we need to discuss this with QA to see if my comments are valid. Otherwise this looks good to me.

Based on the discussion this morning data-test-id named attributes must be present for UI elements.

Reply:
Please add a `content-class`, which is attached to the actual dialog.
Do not change what you've done for `data-test-id`.